.github/workflows/docs-ci.yaml

```diff
@@ -0,0 +1,29 @@
+name: docs-ci
+on:
+  push:
+    branches:
+      - master
+      - main
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Configure Git Credentials
+        run: |
+          git config user.name github-actions[bot]
```

Comment:
Do you have to configure this at all?

Reply:
https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions 

According to this it is needed. I think it's so github actions is able to publish to pages.